pages/index.js

```diff
@@ -16,16 +18,19 @@ export default function HomePage({ homePageData }) {
       {renderRichText(featuredStoriesTitle)}
       {renderRichText(newsletterSignup)}
       {renderRichText(newsletterDescription)}
+      <Footer footerData={footerData}/>
```

Comment:
Should probably put it in the _app.js file so that it can be across all the pages

Reply:
Nevermind! We discussed it here: https://github.com/Scout-NU/thealleyway-site/pull/20#discussion_r595574938